fire-select.js

```diff
@@ -143,7 +143,11 @@ Vue.component('fire-select', {
             this.singleDeselect();
 
             item.selected = true;
-            if (! this.isPopulating) this.$dispatch('fsItemSelected', Vue.util.extend({}, item));
+            if (! this.isPopulating) {
+                setTimeout(function() {
+                    this.$dispatch('fsItemSelected', Vue.util.extend({}, item));
+                }.bind(this), 400);
```

Comment:
why setTimeout 400? if you want this to be propagated after DOM is updated, you should use Vue.nextTick, right?